clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -179,8 +178,8 @@ export class CopybookDownloadService implements vscode.Disposable {
     );
     if (!fs.existsSync(copybookPath)) {
       try {
-        // create .gitignore file within .c4z folder
-        createFileWithGivenPath(C4Z_FOLDER, GITIGNORE_FILE, "/**");
+        // create .gitignore file within extension folder
```

Comment:
Isn't this completely redundant now that the files are outside of the workspace?

Reply:
resolved at : 224e9b8e00a6753439dd54d8da5d6df093b6cf89